netty/src/test/java/io/grpc/netty/GrpcHttp2HeadersUtilsTest.java

```diff
@@ -133,6 +133,101 @@ public void decode_emptyHeaders() throws Http2Exception {
     assertThat(decodedHeaders.toString()).contains("[]");
   }
 
+  // contains is used by Netty 4.1.75+. https://github.com/grpc/grpc-java/issues/8981
```

Comment:
Let's make it clearer what is "contains"
```suggestion
  // `contains()` is used by Netty 4.1.75+. https://github.com/grpc/grpc-java/issues/8981
```

Reply:
Oh, definitely. I just added the `()`, not the back-ticks.

I'm not eager to bring markdown syntax into Java, as we already have to deal with normal text vs Javadoc/HTML. I could use double quotes, or the old-school backtick+single-quote, or even markdown backticks, but it seems to be clear without any of them.